python/pyspark/sql/connect/dataframe.py

```diff
@@ -952,6 +952,26 @@ def freqItems(
 
     freqItems.__doc__ = PySparkDataFrame.freqItems.__doc__
 
+    def sampleBy(
+        self, col: "ColumnOrName", fractions: Dict[Any, float], seed: Optional[int] = None
+    ) -> "DataFrame":
+        if not isinstance(col, (Column, str)):
```

Comment:
The behavior is changed from pyspark sql
```
if isinstance(col, str):
    col = Column(col)
```

Reply:
there is no behavior change, since the underlying plan can accept a `ColumnOrName`

Reply:
Got it.